mods/ra/rules/defaults.yaml

```diff
@@ -198,13 +205,13 @@
 		Upgrades: hospitalheal
 		Prerequisites: hosp
 	DeathSounds@NORMAL:
-		DeathTypes: 1, 2, 3, 4
```

Comment:
The upgrade rules should map each death type to a `DeathTypeX` type for all mods.  A second mod-specific set of rules can then map those names to the mod-specific damage types (`Explosion`, `Tiberium`, etc).